lib/src/test/kotlin/starknet/data/TypedDataTest.kt

```diff
@@ -704,6 +706,72 @@ internal class TypedDataTest {
         }
     }
 
+    @Nested
+    inner class JsonConversionTest {
+        private val types = mapOf(
+            "StarknetDomain" to listOf(
+                TypedData.StandardType("name", "shortstring"),
+                TypedData.StandardType("version", "shortstring"),
+                TypedData.StandardType("chainId", "shortstring"),
+                TypedData.StandardType("revision", "shortstring"),
+            ),
+            "Example" to listOf(
+                TypedData.StandardType("n0", "felt"),
+                TypedData.StandardType("n1", "bool"),
+                TypedData.StandardType("n2", "string"),
+                TypedData.StandardType("n3", "selector"),
+                TypedData.StandardType("n4", "u128"),
+                TypedData.StandardType("n5", "i128"),
+                TypedData.StandardType("n6", "ContractAddress"),
+                TypedData.StandardType("n7", "ClassHash"),
+                TypedData.StandardType("n8", "timestamp"),
+                TypedData.StandardType("n9", "shortstring"),
+            ),
+        )
+        private val domainObject = """
+            {
+                "name": "StarkNet Mail",
+                "version": "1",
+                "chainId": "1",
+                "revision": 1
+            }
+        """.trimIndent()
```

Comment:
Is `trimIndent()` necessary here? Why isn't it used in the `messageObject` below?

Reply:
> Is trimIndent() necessary here?

`trimIndent()` is used in every JSON string in `TypedDataTest` and in other tests in our SDK - that's why I wanted to follow this "standard".

> Why isn't it used in the messageObject below?

Didn't notice it - added 🫡